users/views.py

```diff
@@ -62,6 +63,9 @@ def post(self, request):
             if not User.objects.filter(email=email).exists():
                 return JsonResponse({'MESSAGE': 'ACCOUNT_NOT_EXIST'}, status=401)
 
+            if User.objects.get(email=email).is_social == 1:
```

Comment:
.is_social이 booleanfield일 것 같은데 그러면 == 1을 굳이 안 쓰셔도 됩니다.
```suggestion
            if User.objects.get(email=email).is_social :
```

Reply:
예외상황 추가했는데 만료된 토큰을 가지고있지 않아서 유효하지 않은 토큰으로 통일했습니다.
조건문 수정했습니다.